src/mito2/src/cache/file_cache.rs

```diff
@@ -618,32 +618,41 @@ pub enum FileType {
     /// Parquet file.
     Parquet,
     /// Puffin file.
-    Puffin,
+    Puffin(u64),
+}
+
+impl std::fmt::Display for FileType {
```

Comment:
Make it same stype as fmt::Formatter and fmt::Result
```suggestion
impl fmt::Display for FileType {
```